src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -1437,7 +1541,7 @@ def validate_global_constraints(self):
                         f"{type(cursor).__name__} rather than a Loop.")
                 cursor = cursor.loop_body.children[0]
 
-        super(OMPLoopDirective, self).validate_global_constraints()
+        super().validate_global_constraints()
 
```

Comment:
Not something for this PR but I note that this file is now > 1.5K lines.

Reply:
I agree is getting to long, but I think the OMP directive implementation needs a clean up and is easier to do having them all in one file, I would wait before splitting the file.